addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1952,6 +1952,13 @@ registry.TextHighlight = publicWidget.Widget.extend({
         // to block the callback on this first notification for observed items.
         this.observerLock = new Map();
         this.resizeObserver = new window.ResizeObserver(entries => {
+            // Some options, like the popup, trigger a resize after a delay
+            // before the page is saved. This causes the highlights to be added
+            // back to the DOM after the "TextHighlight" widget has been
+            // destroyed. This is why the following line is needed.
+            if (this.isDestroyed()) {
+                return;
+            }
```

Comment:
Nitpicking: Isn't better to have the same diff in the [old observer](https://github.com/odoo/odoo/pull/178022/commits/156abddd42b8bc24e026496898a66e3da83c68ac#diff-83d3114f3623d6979be3f3962e343e0d34d3f3a8f43e086c4d8ecad46bb89160R2037) (kept for compatibility in stable) ?

Reply:
@xO-Tx Yes, good idea I made the change.